tests/unit/test_api.py

```diff
@@ -0,0 +1,33 @@
+
+from unittest import TestCase
+
+import mock
+from group_project_v2.json_requests import GET
+
+from group_project_v2.project_api import api_implementation
+from tests.utils import TestWithPatchesMixin
+
+
+class BaseApiTest(TestCase, TestWithPatchesMixin):
+
+    def request_side_effect(self):
```

Comment:
@jbzdak looks like it is not used anywhere - is there some implicit use somewhere?